community-meeting-info.md

```diff
@@ -0,0 +1,5 @@
+ 
+**Community meeting:** Every alternate Monday starting May 15th, 2023 at 10AM US
```

Comment:
We may change meeting time in the future again, maybe we can maintain it in the agenda doc (considering it is easier to update than making a PR)?

Reply:
I think its ok to leave the time for people to come to the meeting here but we can probably remove the date, the agenda doc is anyways linked so that is another place we can maintain the time, but usually all repos advertise meetings times outside the agenda to make it easier for people to get the info/join in

Reply:
Yeah, that's the point; how will a newcomer find the agenda in the first place ?..

IMO this should be advertised in the repo (and it is, but is currently misleading).

Reply:
> Yeah, that's the point; how will a newcomer find the agenda in the first place ?..
> 
> IMO this should be advertised in the repo (and it is, but is currently misleading).

Reply:
> Yeah, that's the point; how will a newcomer find the agenda in the first place ?..
> 
> IMO this should be advertised in the repo (and it is, but is currently misleading).

**EDIT**: Having said that, I'm also fine w/ just removing the wrong info, and linking the notes, indicating people to check when the last meeting was, and add 2 weeks to it.

Your call.

Reply:
@npinaeva merging this hope that's fine, this is not a big deal IMHO, we can fix it later if needed.